src/dom_components/view/ComponentTextView.ts

```diff
@@ -182,13 +182,13 @@ export default class ComponentTextView extends ComponentView<ComponentText> {
       const offset = range.startOffset;
       const textModel = getComponentModel(textNode);
       const newCmps: (ComponentDefinition | Component)[] = [];
-
+      let data = textNode.textContent ?? '';
```

Comment:
suggestion: `const data = textNode.textContent || '';`, because the `data` is not updated afterwards.

Reply:
done